mitosheet/mitosheet/api/get_completion.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+import json
+from typing import Any, Dict, Optional
+from mitosheet.types import Selection, StepsManagerType
+import os
+
+from mitosheet.ai.prompt import get_prompt
+
+
+import requests
+
+URL = 'https://api.openai.com/v1/completions'
+
+def get_completion(params: Dict[str, Any], steps_manager: StepsManagerType) -> str:
+        selection: Optional[Selection] = params['selection']
+        user_input: str = params['user_input']
+
+        prompt = get_prompt(
+                steps_manager.curr_step.final_defined_state.df_names,
+                steps_manager.curr_step.final_defined_state.dfs,
+                selection,
+                user_input
+        )
+
+        OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
+
+        if OPENAI_API_KEY is None:
+                return json.dumps({
+                        'error': 'Please aquire an OPENAI_API_KEY and set it as an enviornmental variable.'
```

Comment:
aquire -> acquire
enviornmental -> environmental

It would be nice if we could point them to some documentation on how to do this. Since a lot of our users are new to Python, this is probably not enough guidance.

Reply:
Yep, we can do this on the frontend!